packages/medusa/src/api/routes/admin/publishable-api-keys/index.ts

```diff
@@ -80,14 +80,61 @@ export default (app) => {
   )
 }
 
+/**
+ * @schema AdminPublishableApiKeysRes
+ * type: object
+ * properties:
+ *   publishable_api_key:
+ *     $ref: "#/components/schemas/PublishableApiKey"
+ */
 export type AdminPublishableApiKeysRes = {
   publishable_api_key: PublishableApiKey
 }
+
+/**
+ * @schema AdminPublishableApiKeysListRes
+ * type: object
+ * properties:
+ *   publishable_api_keys:
+ *     type: array
+ *     items:
+ *       $ref: "#/components/schemas/PublishableApiKey"
```

Comment:
The OAS is missing paginated list fields (limit, count, etc...) because it was originally missing them

Reply:
Fixed in 210567146c4a65233d2c3c8e6344fdee450c3e0f